src/Libraries/RevitNodes/Elements/Parameter.cs

```diff
@@ -25,12 +32,216 @@ public string Name
             }
         }
 
+        /// <summary>
+        /// Get the value of the parameter
+        /// </summary>
+        public object Value
+        {
+            get
+            {
+                switch (this.InternalParameter.StorageType)
```

Comment:
Sorry if I am too nagging! But it will be great if you can wrap similar code from Element and this class into some utility methods!